{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"invalidRequest"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/oauth2\/responseerror\/errortype\/invalidrequest"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/OAuth2\/documentation\/OAuth2\/ResponseError\/ErrorType\/invalidRequest","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"The request is missing a required parameter, includes an invalid parameter value,"},{"type":"text","text":" "},{"type":"text","text":"includes a parameter more than once, or is otherwise malformed."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"invalidRequest"}],"title":"ResponseError.ErrorType.invalidRequest","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:6OAuth213ResponseErrorV0C4TypeO14invalidRequestyA2EmF","modules":[{"name":"OAuth2"}]},"hierarchy":{"paths":[["doc:\/\/OAuth2\/documentation\/OAuth2","doc:\/\/OAuth2\/documentation\/OAuth2\/ResponseError","doc:\/\/OAuth2\/documentation\/OAuth2\/ResponseError\/ErrorType"]]},"references":{"doc://OAuth2/documentation/OAuth2/ResponseError/ErrorType/invalidRequest":{"role":"symbol","title":"ResponseError.ErrorType.invalidRequest","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"invalidRequest"}],"abstract":[{"type":"text","text":"The request is missing a required parameter, includes an invalid parameter value,"},{"type":"text","text":" "},{"type":"text","text":"includes a parameter more than once, or is otherwise malformed."}],"identifier":"doc:\/\/OAuth2\/documentation\/OAuth2\/ResponseError\/ErrorType\/invalidRequest","kind":"symbol","type":"topic","url":"\/documentation\/oauth2\/responseerror\/errortype\/invalidrequest"},"doc://OAuth2/documentation/OAuth2/ResponseError/ErrorType":{"role":"symbol","title":"ResponseError.ErrorType","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ErrorType"}],"abstract":[{"type":"text","text":"Errors described in"},{"type":"text","text":" "},{"type":"reference","isActive":true,"identifier":"https:\/\/datatracker.ietf.org\/doc\/html\/rfc6749#section-4.1.2.1"}],"identifier":"doc:\/\/OAuth2\/documentation\/OAuth2\/ResponseError\/ErrorType","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ErrorType"}],"url":"\/documentation\/oauth2\/responseerror\/errortype"},"https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1":{"title":"4.1.2.1. Error Response","titleInlineContent":[{"type":"text","text":"4.1.2.1. Error Response"}],"type":"link","identifier":"https:\/\/datatracker.ietf.org\/doc\/html\/rfc6749#section-4.1.2.1","url":"https:\/\/datatracker.ietf.org\/doc\/html\/rfc6749#section-4.1.2.1"},"doc://OAuth2/documentation/OAuth2":{"role":"collection","title":"OAuth2","abstract":[{"type":"text","text":"An OAuth2 client to authenticate your app with a service provider."}],"identifier":"doc:\/\/OAuth2\/documentation\/OAuth2","kind":"symbol","type":"topic","url":"\/documentation\/oauth2"},"doc://OAuth2/documentation/OAuth2/ResponseError":{"role":"symbol","title":"ResponseError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"ResponseError"}],"abstract":[{"type":"text","text":"Error response to an authorization request."}],"identifier":"doc:\/\/OAuth2\/documentation\/OAuth2\/ResponseError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ResponseError"}],"url":"\/documentation\/oauth2\/responseerror"}}}